modules/installation-registry-osp-creating-custom-pvc.adoc

```diff
@@ -0,0 +1,134 @@
+//
+// * registry/configuring_registry_storage/configuring-registry-storage.adoc
+
+[id="installation-registry-osp-creating-custom-pvc_{context}"]
+= Configuring an image registry with a custom back end on clusters that run on {rh-openstack}
+
+After you install a cluster on {rh-openstack-first}, you can use a Cinder volume that is in a specific availability zone for registry storage.
+
+.Procedure
+
+. Create a YAML file that specifies a storage class that includes the availability zone that you want to use. For example:
++
+[source,yaml]
+----
+apiVersion: storage.k8s.io/v1
+kind: StorageClass
+metadata:
+  name: custom-csi-storageclass
+provisioner: cinder.csi.openstack.org
+volumeBindingMode: WaitForFirstConsumer
+allowVolumeExpansion: true
+parameters:
+  availability: <availability_zone_name>
+----
++
+[NOTE]
+====
+{product-title} does not verify the existence of the availability zone you choose. Verify the name of the availability zone before you apply the configuration.
+====
+
+. From a command line, apply the configuration:
++
+.Example input
+[source,terminal]
+----
+$ oc apply -f <storage_class_file_name>
+----
++
+.Example output
+[source,terminal]
+----
+storageclass.storage.k8s.io/custom-csi-storageclass created
+----
+
+. Create a YAML file that specifies a persistent volume claim (PVC) that uses your storage class and the `openshift-image-registry` namespace. For example:
++
+[source,yaml]
+----
+apiVersion: v1
+kind: PersistentVolumeClaim
+metadata:
+  name: csi-pvc-imageregistry
+  namespace: openshift-image-registry <1>
+  annotations:
+    imageregistry.openshift.io: "true"
+spec:
+  accessModes:
+  - ReadWriteOnce
+  volumeMode: Block
+  resources:
+    requests:
+      storage: 100Gi <2>
+  storageClassName: <your_custom_storage_class> <3>
+----
+<1> Enter the namespace `openshift-image-registry`. This namespace allows the Cluster Image Registry Operator to consume the PVC.
+<2> Optional: Adjust the volume size.
+<3> Enter the name of the storage class that you created. 
+
+. From a command line, apply the configuration:
++
+.Example input
+[source,terminal]
+----
+$ oc apply -f <pvc_file_name>
+----
++
+.Example output
+[source,terminal]
+----
+persistentvolumeclaim/csi-pvc-imageregistry created
+----
+
+. Replace the original persistent volume claim in the image registry configuration with the new claim:
++
+.Example input
+[source,terminal]
+----
+$ oc patch configs.imageregistry.operator.openshift.io/cluster --type 'json' -p='[{"op": "replace", "path": "/spec/storage/pvc/claim", "value": "csi-pvc-imageregistry"}]'
+----
++
+.Example output
+[source,terminal]
+----
+config.imageregistry.operator.openshift.io/cluster patched
+----
++
+Over the next several minutes, the configuration is updated.
+
+. Verify that the registry is using the resources that you defined:
+.. Verify that the PVC claim value is identical to the name that you provided in your PVC definition:
++
+.Example input
```

Comment:
This `.Example input` block title use is new to me.  I don't see what benefit it adds for our customers.